src/main/java/org/folio/fqm/service/EntityTypeService.java

```diff
@@ -153,13 +154,15 @@ private ColumnValues getTenantIds(EntityType entityType) {
         .toList();
       return new ColumnValues().content(tenantValues);
     } else {
-      // tenant is not central tenant, but we need to provide central tenant id as an available value
-      String centralTenantId = crossTenantQueryService.getCentralTenantId();
       List<ValueWithLabel> tenantList = new ArrayList<>();
-      if (centralTenantId != null && !centralTenantId.equals(tenants.get(0))) {
-        tenantList.add(new ValueWithLabel().value(centralTenantId).label(centralTenantId));
-      }
       tenantList.add(new ValueWithLabel().value(tenants.get(0)).label(tenants.get(0)));
+      if (SIMPLE_INSTANCES_ID.equals(entityType.getId())) {
+        // tenant is not central tenant, but we need to provide central tenant id as an available value
+        String centralTenantId = crossTenantQueryService.getCentralTenantId();
+        if (centralTenantId != null && !centralTenantId.equals(tenants.get(0))) {
+          tenantList.add(new ValueWithLabel().value(centralTenantId).label(centralTenantId));
+        }
+      }
```

Comment:
Please don't hate me. We need special functionality for the tenant ID field only for instances, and there's no real way to do that without either checking the entity type ID or adding another one-off property.